### PR TITLE
Remove apply now section

### LINKED ISSUE #1486: Remove the apply now section from the join us page
**Is your feature request related to a problem? Please describe.**
The join us page has a section as [Apply Now](https://sefglobal.org/join-us.html). We no longer require that section.

**Describe the solution you'd like**
Remove the `apply now` section.

**Describe alternatives you've considered**
None

**Additional context**
Screenshot 2022-12-26 at 18 03 10

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by Pirijatharsan**
## Purpose

The purpose of this PR is to fix #1486

## Goals

The join us page has a section as [Apply Now](https://sefglobal.org/join-us.html). We no longer require that section.

## Approach

Remove the apply now section

### Screenshots

image

 
### Preview Link

https://pr-1487-sef-site.surge.sh/join-us.html

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

## Related PRs
 none

## Learning 

none

### CODE REVIEW

**Comment by Madhawa97** [comment]
Good job @piriyatharsan. 
@anjula-sack It seems we don't have any CSS code specific to this section. so this is safe to merge right?

File: join-us.html [hunk 1]
```diff
@@ -189,26 +189,6 @@ <h1 class="display-3 text-center mb-5 mt-4">Frequently Asked Questions</h1>
...
-        <div class="container">
-            <div class="row">
-                <div class="col-md-6">
-                    <img src="assets/img/join_us/img2.svg">
-                </div>
-                <div class="col-md-6 mt-auto mb-auto pl-5">
-                    <div class="display-3 font-weight-light" style="color: #3d317c">Join the <span
-                            class="font-weight-bold d-inline">Team</span></div>
-                    <p class="lead mt-auto mb-auto">I want to join the SEF Team</p>
-                    <a href="apply.html">
```
Comment (anjula-sack) [inline join-us.html:203]: I think we can remove the apply.html file as well

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: join-us.html [hunk 2]
```diff
@@ -189,26 +189,6 @@ <h1 class="display-3 text-center mb-5 mt-4">Frequently Asked Questions</h1>
         </div>
     </section>
```
Comment (YoshithaRathnayake) [inline join-us.html]: You can remove this line also @piriyatharsan 
```suggestion
```

**Review by YoshithaRathnayake — DISMISSED** [review]

File: join-us.html (hunk 2 above)
Comment (YoshithaRathnayake) [reply join-us.html]: @piriyatharsan can you do the above mentioned changes?

File: join-us.html (hunk 1 above)
Comment (Piumal1999) [reply join-us.html:203]: Shall we just merge this Pull request and create another issue for removing the `apply.html` file?

Comment (anjula-sack) [reply join-us.html:203]: yeah I'll create one @Piumal1999

**Review by anjula-sack — DISMISSED** [review]
Thank you! @piriyatharsan

**Review by anjula-sack — APPROVED** [review]

**Pull request merged**